src/Carousel/Carousel.xml

```diff
@@ -0,0 +1,194 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="Carousel.widget.Carousel" needsEntityContext="true" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../xsd/widget.xsd">
+    <name>Carousel</name>
+    <description>Display images in a carousel either from static or dynamic data source</description>
+  
+      <icon>iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2Fy
+ZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAfxJREFUeNqkk89LFGEcxp935t3WZmfD
+XWTZRDTYgoTWS3aI6OgxKMRL+w9067A3uxTUOQqCDt50L160qGtdxBAhDx0ETaTW
+UduBmaJ1Z8aZebf3++4Uq7goNPDy/ph5Ps/3eb8Ma7fb+J+HP56fB0EYY3/nO/L8
+4hn1e7x7R5AwDAefVSqvT6uLyfGoVnvQDVDOEnJ7dWsL1lr1BJWUySr78AsTkx9w
+GASM93IeLy32dH73ZgqREPATwBHnFx8ZuAbockQCyPYBKbkezgMD5nncv1kE56kO
+wPMYP+48W+qd+f3aphIeRucQdgAapxyxzPVlodBD1kGX7zVkJbEC7O9aam5RBZSD
+NppmIp0W6p6UVGfQUzpELCBkFvomjn3lbFk+ImnqtVpMoxz08uWrIXxaKcAwDGQy
+GeSKRQxdv4FC6TJMM6sA604HdHXiKXTO4TqOzikHUTfWt2H/6JeHBoZHdFy7NYKB
+bBmB15Ci38q5eRAp5+czM28fTk8vxc2mzVtJBdWqK8v/KaNoqq329ldkLyzBrkdw
+rADlcYEr/Z5yjnz/s728XJNJXU45iLqzI5KLixXAcEKwTIT9bwKN7zFGjzrvpXO5
+3cB1wSkHUe8+sU9s36VkPu78r0/5sbGKbpqDZ/lzyPmgXp8j5+5Gp+XIJ+vTHlJ6
+3Qd/BBgAioQXVN0jZskAAAAASUVORK5CYII=</icon>
+
+     <properties>
+            <property key="imageEntity" type="entity" required="false">
+                <caption>Entity</caption>
+                <category>Data source</category>
+                <description>The entity which should be displayed in the carousel</description>
+            </property>
+            <property key="imageSource" type="enumeration" defaultValue="microflow">
```

Comment:
Microflows should be a last resort as they cannot be optimized. Ideally "database" data sources should be used or XPath.